indico/web/assets/vars_js.py

```diff
@@ -117,6 +117,10 @@ def generate_global_file(config):
             },
 
             'Timetable': {
+                'entry': {
+                    'move': url_rule_to_js('timetable.move_timetable_entry'),
+                    'changeDates': url_rule_to_js('timetable.change_start_date')
```

Comment:
I don't think `end_dt` (which is calculated anyway) changing due to a duration change is something worth displaying as an auto-op. *Obviously* changing the duration of something changes the end time ;)